javascript/js-array-methods-2_animal-objects/index.js

```diff
@@ -91,22 +91,23 @@ const indexOfAnimalWithNameLongerFive = null;
 // Hint: sort() mutates the original array, which is bad.
 // -> Use animals.slice().sort(...) to make a copy (and the tests work).
 
-const animalsSortedAlphabetically = null;
+const animalsSortedAlphabetically = animals.slice().sort((a, b) => a.name < b.name ? -1 : 1);
 
-const animalsSortedByWeightStartingWithLowest = null;
+const animalsSortedByWeightStartingWithLowest = animals.slice().sort((a, b) => a.weight < b.weight ? -1 : 1);
```

Comment:
When they are number, you can just return the value of `a-b`

Reply:
Thanks for the tip 🫶🏼